lib/proto/compact_formats.proto

```diff
@@ -37,20 +37,30 @@ message CompactTx {
     //    valueBalance + (sum(vPubNew) - sum(vPubOld) - sum(tOut))
     uint32 fee = 3;
 
-    repeated CompactSpend spends = 4;   // inputs
-    repeated CompactOutput outputs = 5; // outputs
+    repeated CompactSaplingSpend spends = 4;   // inputs
+    repeated CompactSaplingOutput outputs = 5; // outputs
+    repeated CompactOrchardAction actions = 6;
 }
 
-// CompactSpend is a Sapling Spend Description as described in 7.3 of the Zcash
+// CompactSaplingSpend is a Sapling Spend Description as described in 7.3 of the Zcash
 // protocol specification.
-message CompactSpend {
+message CompactSaplingSpend {
     bytes nf = 1;   // nullifier (see the Zcash protocol specification)
 }
 
 // output is a Sapling Output Description as described in section 7.4 of the
 // Zcash protocol spec. Total size is 948.
-message CompactOutput {
+message CompactSaplingOutput {
     bytes cmu = 1;          // note commitment u-coordinate
     bytes epk = 2;          // ephemeral public key
-    bytes ciphertext = 3;   // ciphertext and zkproof
+    bytes ciphertext = 3;   // first 52 bytes of ciphertext
+}
+
+// https://github.com/zcash/zips/blob/main/zip-0225.rst#orchard-action-description-orchardaction
+// (but not all fields are needed)
```

Comment:
So again...   omit as many fields as possible to be `Compact`?

Reply:
Yeah, that's my impression.